pkg/detectors/kanbantool/kanbantool.go

```diff
@@ -0,0 +1,85 @@
+package kanbantool
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"kanbantool"}) + `\b([0-9A-Z]{12})\b`)
+	domainPat = regexp.MustCompile(detectors.PrefixRegex([]string{"kanbantoolDomain"}) + `\b([a-z0-9A-Z._]{4,22})\b`)
```

Comment:
suggestion: After some testing, they allow a minimum of 2 characters and maybe we can go with `\b([a-z0-9A-Z._]{2,22})\b`?